README.md

```diff
@@ -44,6 +44,25 @@ $ bundle exec rubocop
 
 You do not need to include rubocop directly in your application's dependencies. Datarockets-style will include a specific version of `rubocop` and `rubocop-rspec` that is shared across all projects.
 
+### Configurable cops
+
+There are some areas in which there is no clear consensus in datarockets team regarding a particular style (like string literal quoting).
+In such scenarios, all popular styles are acknowledged and it’s up to you to pick one and apply it consistently.
+For that just set up these cops before starting of usage.
+
+#### Style/StringLiterals
+
+There are two popular styles in the Ruby community, both of which are considered good - single quotes by default and double quotes by default.
+There are not any clear consensus as in the Ruby community, as in the datarockets team about this style.
```

Comment:
`There is no clear consensus about this style in the Ruby community and in the datarockets team`